### PR TITLE
feat(login): Add password reset link to login window

### PR DESCRIPTION
**Pull request by kvviecien**
When user gets an error while logging in, e.g. when putting in incorrect credentials, link to password reset page appears on the bottom.

Issue: #416
---
Also changed the position and styling of the error message and and the checkboxes. Looks simpler, more modern. Screenshots below:

runbox_old 
runbox_new

### CODE REVIEW

**Comment by kvviecien** [comment]
I followed the suggestion of @gtandersen from Issue #416 about showing the link only when the username/password combination is wrong. But I thought maybe it would be better if the link was visible all the time? 

And I am so sorry, I just noticed @hernan604 was already assigned to it. Issue #525 looked very pleasing to me and I thought I would make some steps towards it, not looking at the assignees...

File: src/app/login/login.component.ts [hunk 1]
```diff
@@ -135,7 +135,8 @@ export class LoginComponent implements OnInit {
         if (loginresonseobj.user_status > 0 && loginresonseobj.user_status < 5 && loginresonseobj.error) {
             this.accountExpired = true;
         } else {
-            this.login_error_html = '<p>' + loginresonseobj.message + ': ' + (loginresonseobj.error || 'error') + '</p>';
+            this.login_error_html = '<p>' + (loginresonseobj.error || 'Error occurred') + '</p>';
+            console.log(loginresonseobj);
```
Comment (tadzik) [inline src/app/login/login.component.ts:139]: Leftover debugging code? :)

Comment (kvviecien) [reply src/app/login/login.component.ts]: Thanks. Got rid of it :)

**Comment by gtandersen** [comment]
Nice work @kvviecien (and I'm sure @hernan604 doesn't mind :).

I suggest only showing the link when the login fails as it takes up another line of vertical space, so the login box gets somewhat cramped depending on the window size.

**Comment by kvviecien** [comment]
Thanks! It's good to go then :)

**Pull request merged**